backend/api/Services/RobotService.cs

```diff
@@ -28,5 +40,32 @@ public async Task<IEnumerable<Robot>> ReadAll()
         {
             return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id, StringComparison.Ordinal));
         }
+
+        public async Task<Report> StartMission(Robot robot, string missionId)
+        {
+            var response = await _isarService.StartMission(missionId);
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new MissionException($"Could not start mission with id: {missionId}");
+            }
+            if (response.Content != null)
+            {
+                string? responseContent = await response.Content.ReadAsStringAsync();
+                var isarResponse = JsonSerializer.Deserialize<IsarStartMissionResponse>(responseContent);
```

Comment:
```suggestion
                var isarResponse = await response.Content.ReadFromJsonAsync<IsarStartMissionResponse>();
```
This works and is slightly smoother.

Reply:
Agreed, fixed